commisery/commit.py

```diff
@@ -24,7 +24,7 @@
     r"""
         # 1. Commits MUST be prefixed with a type, which consists of a noun, feat, fix, etc.,
         # followed by the OPTIONAL scope, OPTIONAL !, and REQUIRED terminal colon and space.
-        (?P<type>\w+)?
+        (?P<type>[\[\]\-\w]+)?
```

Comment:
Why make the conventional commit regex match things that are not conventional commits? Why don't you just handle the fact that this regex doesn't match and then give an error that it's not a conventional commit (when that's what is desired)?

Reply:
As discussed, agreed to keep the previous regex as this will lead to meaningful error messages in the first place.